emva1288/unittests/test_results.py

```diff
@@ -300,3 +302,68 @@ def test_nans(self):
         self.assertIs(r.DSNU1288_DN(), np.nan)
 
         del r
+
+
+class TestRoutines(unittest.TestCase):
+
+    # instantiate a bayer filter and a wrong filter
+    bayer_filter = get_bayer_filter(0, 1, 1, 0, 1000, 1000)
+    wrong_filter = np.tile([[1, 1, 1, 0], [1, 0, 0, 1]], (500, 250))
+
+    # intantiate masked images based in the masks
+    img = np.abs(np.random.random([1000, 1000]))
+    wrong = np.ma.array(img, mask=wrong_filter)
```

Comment:
name them as img_wrong, and img_masked, we will know they are the "source" images

Reply:
Changed wrong as suggested, but instead of changing masked to img_masked, i changed it to imgc (for color image). It represents well what we try to achieve, and it makes the code a lot easier to read, in my opinion.